src/components/Notification/InactivityMessage.jsx

```diff
@@ -58,25 +61,54 @@ const InactivityMessage = () => {
     };
   }, []);
 
+  /* Starts a five minute timer when the inactivity window pops up. 
+     Logs the user out and refreshes the page if time elapses. */
+  useEffect(() => {
+    if (showPopup) {
+      logoutTimer.current = setInterval(() => {
+        if (seconds === 0 && minutes === 0) {
+          handleLogout();
```

Comment:
I believe the linter would pick this up, it looks like `handleLogout` is used before it's defined. Should be able to fix this by moving `handleLogout` before this `useEffect`.

Reply:
I moved the handleLogout function up as you suggested. You mentioned before about the branch starting from an older version of Development. Does that mean I should start a new branch to put these changes into?

Reply:
You could do that, but you don't have to. You simply need to update this branch by merging the changes made in Development into this one. But because they're from diverging branches, it would have to deal with merge conflicts.

Reply:
If you were to merge development into this branch you would have to re base off of Development. Re basing should resolve the merge conflicts. I Updated instructions in contributing.md because previous instructions did not specify what to base from when creating a branch on GitHub issues. This looks like it was created through that and by default based off of Master.